index.html

```diff
@@ -1,41 +1,249 @@
+          </h3>
+          <p class="recentWorkDesc wc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="usedTechs wc">
+            <li class="tech twc">HTML</li>
+            <li class="tech twc">Bootstrap</li>
+            <li class="tech twc">Ruby</li>
+          </ul>
+        </div>
+        <button class="btn spwc" type="button">See Project</button>
+      </div>
+      <div class="workCard">
+        <div class="workCardContent">
+          <h3 class="recentWorkTitle wc">
+            Profesional Art <br />Printing Data
+          </h3>
+          <p class="recentWorkDesc wc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="usedTechs wc">
+            <li class="tech twc">HTML</li>
+            <li class="tech twc">Bootstrap</li>
+            <li class="tech twc">Ruby</li>
+          </ul>
+        </div>
+        <button class="btn spwc" type="button">See Project</button>
+      </div>
+      <div class="workCard">
+        <div class="workCardContent">
+          <h3 class="recentWorkTitle wc">
+            Profesional Art <br />Printing Data
+          </h3>
+          <p class="recentWorkDesc wc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="usedTechs wc">
+            <li class="tech twc">HTML</li>
+            <li class="tech twc">Bootstrap</li>
+            <li class="tech twc">Ruby</li>
+          </ul>
+        </div>
+        <button class="btn spwc" type="button">See Project</button>
+      </div>
+      <div class="workCard">
+        <div class="workCardContent">
+          <h3 class="recentWorkTitle wc">
+            Profesional Art <br />Printing Data
+          </h3>
+          <p class="recentWorkDesc wc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="usedTechs wc">
+            <li class="tech twc">HTML</li>
+            <li class="tech twc">Bootstrap</li>
+            <li class="tech twc">Ruby</li>
+          </ul>
+        </div>
+        <button class="btn spwc" type="button">See Project</button>
+      </div>
+    </section>
+    <section class="aboutme">
+      <h3 class="aboutmeTitle">About me</h3>
+      <p class="aboutmeDesc recentWorkDesc wc">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button class="btn getResume" type="button">Get My Resume</button>
```

Comment:
- [ ] It seems that this button as well, has lost the `background-color`,  let us fix this please 😉 
![image](https://user-images.githubusercontent.com/55465133/130125082-b9456c39-bb36-40e4-8835-ff6c0cdf288d.png)

Reply:
It is weird because the buttons has the color and is working fine.